solution/ui/regulations/eregs-vite/src/components/policy-repository/SubjectSelector.vue

```diff
@@ -15,42 +20,140 @@ const props = defineProps({
 const $router = useRouter();
 const $route = useRoute();
 
+const state = reactive({
+    filter: "",
+    subjects: [],
+});
```

Comment:
new local state for subject filter querystring and the array of filtered subjects.  This will not be stored in the URL